repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -61,14 +61,14 @@ public Launcher(String[] args) throws JSAPException {
         JSAP jsap = this.defineArgs();
         JSAPResult arguments = jsap.parse(args);
         LauncherUtils.checkArguments(jsap, arguments, LauncherType.SCANNER);
+        this.initConfig(arguments);
```

Comment:
Could you add a comment here to specify that "this.config" will be available only after this call. It might avoid future NPE here.

Reply:
Done.